JetStreamCompose/jetstream/src/main/java/com/google/jetstream/presentation/utils/ModifierUtils.kt

```diff
@@ -78,6 +83,39 @@ fun Modifier.handleDPadKeyEvents(
     false
 }
 
+/**
+ * Handles all D-Pad Keys
+ * */
+fun Modifier.handleDPadKeyEvents(
```

Comment:
this is called the same way as the other `handleDPadKeyEvents` but it uses different modifier. Should we rename the other one? Or what's the difference between them actually?